### DISCUSSION TITLE
Beatmap Restrictions in Multiplayer Lobbies

### DISCUSSION THREAD

**Discussion by tylort69** [ppy/osu#17959] [Ideas] [answered]
Having a simple way to moderate the kinds of Beatmaps added to the queue in multiplayer lobbies would be a really nice feature.
This would prevent people from being able to add extremely difficult maps to lobbies filled with new players and ruining their experience of the game. It would also make it much easier to have a DT only lobby or anything specific like that.
I'm not a UI designer so take the example with a grain of salt its more to ensure what I'm asking for is understood.
![box not checked](https://user-images.githubusercontent.com/28652424/164967563-cb6784eb-c199-4d30-9b77-d4e190b2dbcc.png)
![Box Checked](https://user-images.githubusercontent.com/28652424/164967569-2d6f1601-f7ac-4e27-8979-15021ceac31b.png)

**Comment by bdach** [comment]
i swear there used to be another discussion for this but i can't find it...

**Comment by integerrr** [comment, marked as answer]
#16790?

**Reply by bdach** [reply]
yeah there we go, thanks.
